### PR TITLE
Privacy Policy (cont.) up2date

### LINKED ISSUE #24: [Privacy Issue] Add a disclaimer about use of Google Analytics
According to [this](https://support.google.com/analytics/answer/7318509?hl=en) Google support article, Google says:
"Privacy Disclosures Policy

When you use Google Analytics on your site or application, you must disclose the use of Google Analytics and how it collects and processes data."

I would recommend that you update your website to comply with this policy and privacy laws, including the GDPR.

### PR DESCRIPTION
**Pull request by GrahamSH-LLK**
Continuation of #27 , resolves #24.

@WorldLanguages, you do the merge as you are the project leader. Up to date version of #27.

### CODE REVIEW

File: site/_includes/footer.html [hunk 1]
```diff
@@ -14,6 +14,7 @@ <h3>Pages</h3>
 			<a href="/faq" rel="noopener">FAQ</a>
 			<a href="/feedback" rel="noopener">Send Feedback</a>
 			<a href="/contributors" rel="noopener">Contributors</a>
+					<a href="/privacy">Privacy Policy</a>
```
Comment (wgyt735yt) [inline site/_includes/footer.html:17]: We should add a rel="noopener" attribute here

**Review by wgyt735yt — CHANGES_REQUESTED** [review]
Everything except footer.html is fine, we just need to add a rel="noopener" attribute to the privacy policy link.

File: site/_includes/footer.html (hunk 1 above)
Comment (GrahamSH-LLK) [reply site/_includes/footer.html]: Added!

**Review by Hans5958 — CHANGES_REQUESTED** [review]
I was going to do this but I'm uncertaim of the compability of this PR. Stand by.

**Review by wgyt735yt — APPROVED** [review]
Thanks for fixing the noopener issue, this seems good to go.

**Comment by Hans5958** [comment]
Closed in favour of #27.

**Pull request closed without merging**